cloudberry/zion/src/main/scala/edu/uci/ics/cloudberry/zion/model/schema/Query.scala

```diff
@@ -15,15 +15,23 @@ trait IWriteQuery extends IQuery {
 
 }
 
-case class QueryExeOption(sliceMills: Int,
-                          continueSeconds: Int)
+case class JsonRequestOption(sliceMills: Int,
```

Comment:
I think this is still the `QueryExeOption`. It is the parsed object result, like `Query`, not the `JSON`, right?